### PR TITLE
VTCCa-4535 added client details to propertiesView

### PR DESCRIPTION
**Pull request by habeeb01**
https://jira.tools.tax.service.gov.uk/browse/VTCCA-4535
https://jira.tools.tax.service.gov.uk/browse/VTCCA-4536

both tickets were implemented in this branch so that we can be sure that they end up in the same release

### CODE REVIEW

**Comment by darrell-plant** [comment]
LGTD 👍

File: app/models/PropertiesView.scala [hunk 1]
```diff
@@ -33,7 +34,8 @@ case class PropertiesView(
       address: Option[String],
       NDRListValuationHistoryItems: Seq[APIValuationHistory],
       parties: Seq[APIParty],
-      agents: Option[Seq[LegacyParty]]) {
+      agents: Option[Seq[LegacyParty]],
```
Comment (darrell-plant) [inline app/models/PropertiesView.scala:37]: I'm not sure about the naming of `LegacyParty`, but that's for another day

Comment (habeeb01) [reply app/models/PropertiesView.scala:37]: Yep, I thought the same when I saw it 😄

**Comment by reddy-yattapu** [comment]
👍

**Pull request merged**